AUTHORS.txt

```diff
@@ -120,3 +120,4 @@
 * `Yaşar Arabacı <https://github.com/yasar11732>`_
 * `Zhaojun Meng <https://github.com/zhaojunmeng>`_
 * `小明 <https://github.com/dongweiming>`_
+* `h4ckninja <https://github.com/h4ckninja>`_
```

Comment:
Ok, one little thing: you should put yourself at the correct (alphabetically sorted) place ;-) But we (or at least Kwpolska) can do that during merging.